src/OpenTelemetry/README.md

```diff
@@ -84,6 +84,51 @@ using var otel = Sdk.CreateTracerProvider(b => b
   * [Building your own Processor](../../docs/trace/extending-the-sdk/README.md#processor)
   * [Building your own Sampler](../../docs/trace/extending-the-sdk/README.md#sampler)
 
+### Troubleshooting
+
+Self diagnostics module is provided to help troubleshooting.
+
+To enable self diagnostics, go to the current directory of your process and
+create a configuration file named `OTEL_DIAGNOSTICS.json` with the following
+content:
+
+```json
+{
+    "LogDirectory": ".",
+    "FileSize": 1024,
```

Comment:
FileSizeInKiloBytes

Reply:
Shall I use FileSizeIn[KibiBytes](https://en.wikipedia.org/wiki/Kibibyte) to be more aligned with the standards? 

Although I can imagine many people might just type in "Kilo" resulting the configuration file as invalid.

Reply:
lets leave it as is  in this PR and have this merged. And come back to this.